### PR TITLE
[2/N] --game_options param for games (used by Ludii)

### PR DESCRIPTION
**Pull request by DennisSoemers**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Allows easy loading of variants of games through Ludii's options mechanism. For example, `--game_name="LudiiHex.lud"` just loads normal Hex, but `--game_name="LudiiHex.lud" --game_options "Board Size/13x13" "End Rules/Misere"` loads Hex on a 13x13 board with Misere rules.

I imagine the same mechanism could also be useful for non-Ludii games later on.

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [ ] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: src/core/game.h [hunk 1]
```diff
@@ -246,9 +247,18 @@ to look into this) if the strategy is identical to knuth’s.
       JNIEnv* jni_env = Ludii::JNIUtils::GetEnv();
 
       if (jni_env) {
-        Ludii::LudiiGameWrapper game_wrapper(ludii_name);
-        state_ =
+		if (gameOptions.size() > 0) {
+		  Ludii::LudiiGameWrapper game_wrapper(ludii_name, gameOptions);
+		  for (const std::string option : gameOptions) {
+	        std::cout << "Using Game Option: " << option << std::endl;
```
Comment (tscmoo) [inline src/core/game.h:253]: Could you just clang-format this file?

Comment (DennisSoemers) [reply src/core/game.h]: done :)

**Review by tscmoo — APPROVED** [review]
Great!

**Comment by tscmoo** [comment]
Merged in https://github.com/facebookincubator/Polygames/commit/634dcc039b0ee1dd6e8726ba84503943271cde33

**Pull request closed without merging**